src/System.Security.Cryptography.Encoding/tests/AsnEncodedDataCollection.cs

```diff
@@ -0,0 +1,94 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Text;
+using System.Linq;
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.Security.Cryptography.Encoding.Tests
+{
+    public static class AsnEncodedDataCollectionTests
+    {
+        [Fact]
+        public static void Nullary()
+        {
+            AsnEncodedDataCollection c = new AsnEncodedDataCollection();
+            AssertEquals(c, Array.Empty<AsnEncodedData>());
+        }
+
+        [Fact]
+        public static void Oneary()
+        {
+            AsnEncodedData a0 = new AsnEncodedData("1.0", Array.Empty<byte>());
+            AsnEncodedDataCollection c = new AsnEncodedDataCollection(a0);
+            AssertEquals(c, new AsnEncodedData[] { a0 });
+        }
+
+        [Fact]
+        public static void Add()
+        {
+            AsnEncodedData a0 = new AsnEncodedData("1.0", Array.Empty<byte>());
+            AsnEncodedData a1 = new AsnEncodedData("1.1", Array.Empty<byte>());
+            AsnEncodedData a2 = new AsnEncodedData("1.2", Array.Empty<byte>());
+
+            AsnEncodedDataCollection c = new AsnEncodedDataCollection();
+            int index;
+            index = c.Add(a0);
+            Assert.Equal(0, index);
+            index = c.Add(a1);
+            Assert.Equal(1, index);
+            index = c.Add(a2);
+            Assert.Equal(2, index);
```

Comment:
There's no need to store into a temporary, e.g. instead of:
```
            int index;
            index = c.Add(a0);
            Assert.Equal(0, index);
            index = c.Add(a1);
            Assert.Equal(1, index);
            index = c.Add(a2);
            Assert.Equal(2, index);
```
it can be just:
```
            Assert.Equal(0, c.Add(a0));
            Assert.Equal(1, c.Add(a1));
            Assert.Equal(2, c.Add(a2));
```